cmake/deps.txt

```diff
@@ -43,4 +43,4 @@ b64;https://github.com/libb64/libb64/archive/refs/tags/v2.0.0.1.zip;815b6d31d50d
 pthread;https://sourceforge.net/projects/pthreads4w/files/pthreads4w-code-v3.0.0.zip;3b9e417e4474c34542b76ad40529e396ac109fb4
 triton;https://github.com/triton-inference-server/server/archive/refs/tags/v2.28.0.zip;4b305570aa1e889946e20e36050b6770e4108fee
 # above are deps introduced by triton client, might remove after 1.14 release
-extensions;https://github.com/microsoft/onnxruntime-extensions/archive/81e7799c69044c745239202085eb0a98f102937b.zip;d53487035174a046628359289ad27aa0ac0380c9
\ No newline at end of file
+extensions;https://github.com/microsoft/onnxruntime-extensions/archive/81e7799c69044c745239202085eb0a98f102937b.zip;5908b8684f34b1176691f6bbe6fec340848d8e5d
```

Comment:
Check if the file uploaded to Azure DevOps Artifacts is the same as the one https://github.com/microsoft/onnxruntime-extensions/archive/81e7799c69044c745239202085eb0a98f102937b.zip

Reply:
I checked using download artifact and get the sha256 is the same.

Reply:
Where did you see the checksum mismatch error? Does it still exist?